### ISSUE TITLE
UC being detected with 3.4.x but not 3.2.x

### ISSUE THREAD

**Issue by versionDefect** [ultrafunkamsterdam/undetected-chromedriver#1075]
### Discussed in https://github.com/ultrafunkamsterdam/undetected-chromedriver/discussions/1074

As discussed in the discussion I am being detected in versions 3.4.4 but NOT being detected in 3.2.1. I can show my code if you'd like. However, it is very bad. This was my first python project coming from C++ so it is not efficient or well-made at all.  

Something I did find quite odd is that if I opened the browser (using UC with python) and manually typed in my credentials it worked perfectly. But when It did it automatically, even with delays and mouse movements and everything, it detected it. 

The only two things that I can think maybe caused this issue is how chrome is better with headless and maybe UC changed its code to adapt to it better. And or, sending keystrokes to a website on newer versions can be detected

EDIT: if you're having this issue aswell. If you are running withOUT headless you should be able use the most recent version (at the time 110). However, For some reason, Headless does not function with versions 3.2.x on chrome version 110, If you want to run headless, then downgrade to version 109.

**Comment by AlexPaiva** [comment]
What about 3.4.4 with 109?

**Comment by versionDefect** [comment]
> What about 3.4.4 with 109?

I don't think I've tried this combination yet. I'll let you know when I get off work.

**Comment by ultrafunkamsterdam** [comment]
3.4.6 is the current version 😅

**Comment by versionDefect** [comment]
> What about 3.4.4 with 109?

Just tried it. Did not work. 

> 3.4.6 is the current version 😅
Sorry was troubleshooting for like 3 hours so brain was a bit fried lol. For some reason, UC just straight up did not like my code for versions <3.4.3 but 3.4.4 and above (and yes 3.4.6) did RUN my code but was being detected. 

I wouldn't really consider this issue closed honestly.

**Comment by ultrafunkamsterdam** [comment]
As you said. It didn't like your code. And there is no code here ... So this shouldn't be raised as an issue in the first place

**Comment by ultrafunkamsterdam** [comment]
Use the discussion board instead

**Comment by versionDefect** [comment]
> As you said. It didn't like your code. And there is no code here ... So this shouldn't be raised as an issue in the first place

I can post my code if that's what you'd like lol. https://pastebin.com/ULsdwFTH . it's a bit long so I trimmed it to where it's primarily just selenium / UC setup, it should also be noted that it is getting detected with or without headless. everything before line 74 is the setup. after 74 is when it starts to login and get detected.

This is the point where my code would with <3.4.3 but 3.4.4 and above would RUN BUT get detected any time I used Send keys. 

I don't want to sound like an expert in something that I know very little about but, I think that a change in the Send KEYS function could have been changed and accidentally allows for detection. 

Just tryna report an issue. Even in the discussions, other people are having these EXACT issues too....
